api/api.gradle.kts

```diff
@@ -10,5 +10,6 @@ dependencies {
     api(group = "com.google.guava", name = "guava", version = "21.0")
     api(group = "org.fxmisc.easybind", name = "easybind", version = "1.0.3")
     api(group = "org.controlsfx", name = "controlsfx", version = "8.40.11")
-    api(group = "edu.wpi.first.wpilib.networktables.java", name = "NetworkTables", version = "+", classifier = "desktop")
-}
\ No newline at end of file
+    api(group = "edu.wpi.first.ntcore", name = "ntcore-java", version = "+")
+    api(group = "edu.wpi.first.wpiutil", name = "wpiutil-java", version = "+")
```

Comment:
Its actually much deeper then that. It's actually an issue that spans across multiple repos. It's an artifact from needing the combined jars to be published all at the same time.